docs/SDK-GUIDE.md

```
@@ -0,0 +1,980 @@
+verify.set(otp: "123456")
+await Otpless.shared.start(withRequest: verify)
+
+// 4. Every result arrives as OtplessResponse via OtplessResponseDelegate.onResponse,
+//    always hopped onto the main queue. Terminal success is responseType == .ONETAP.
+```
+
+---
+
+## 2. Repository & Source Layout
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)